oneanddone/users/tests/test_models.py

```diff
@@ -31,6 +31,16 @@ def test_attempts_in_progress(self):
         tasks = TaskAttemptFactory.create_batch(4, user=user, state=TaskAttempt.STARTED)
         eq_(set(user.attempts_in_progress), set(tasks))
 
+    def test_bugzilla_url_with_bugzilla_email(self):
+        user = UserProfileFactory.create(bugzilla_email='foo@example.com')
+        activity_url = 'https://bugzilla.mozilla.org/page.cgi?'\
+            'id=user_activity.html&action=run&from=-14d&who=%s' % (user.bugzilla_email)
```

Comment:
The parentheses around `user.bugzilla_email` are unnecessary here as there is only one variable being substituted in the string.

Also, you are kind of duplicating the code for `bugzilla_url` in this test method, which should be avoided. I would suggest doing something like this:

```python
    def test_bugzilla_url_with_bugzilla_email(self):
        email = 'foo@example.com'
        user = UserProfileFactory.create(bugzilla_email=email)
        activity_url = 'https://bugzilla.mozilla.org/page.cgi?'\
            'id=user_activity.html&action=run&from=-14d&who=%s' % email
        eq_(user.bugzilla_url, activity_url)
```